src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/negationRationale/NegationRationale.tsx

```diff
@@ -69,15 +83,17 @@ const NegationRationale = ({
             required={false}
           />
         </div>
-        <div tw="flex-grow py-6">
-          <IconButton
+        <div tw="relative pl-2.5">
+          <Button
+            tw="absolute bottom-0"
             data-testid="add-negation-rationale"
             onClick={() => {
               handleChange(negationRationale);
             }}
+            disabled={isFormValid()}
```

Comment:
Is there any reason why we can't do 
```suggestion
            disabled={_.isEmpty(negationRationale)}
```

Reply:
@RohitKandimalla I changed it thanks!